blueocean-dashboard/src/main/js/components/Branches.jsx

```diff
@@ -12,6 +12,29 @@ import { CellRow, CellLink } from './CellLink';
 
 import { buildRunDetailsUrl } from '../util/UrlUtils';
 
+function noRun(branch, openRunDetails, t, store) {
```

Comment:
IMO it would have make sense to create a `<NoRun/>`component with proper props validation. 

This and the PR one have a lot in common and it would help to treat them exactly the same.